homeassistant/components/zha/strings.json

```
@@ -249,6 +249,13 @@
       "face_4": "With face 4 activated",
       "face_5": "With face 5 activated",
       "face_6": "With face 6 activated"
+    },
+    "extra_fields": {
+      "color": "Color",
+      "duration": "Duration",
+      "effect_type": "Effect type",
+      "led_number": "LED number",
+      "level": "Level"
```

Is this brightness? Then alternatively
```suggestion
      "level": "Brightness"
```
or
```suggestion
      "level": "Brightness (%)"
```

---

It looks like your second suggestion is correct, it is the brightness as a percentage.